cou/steps/plan.py

```diff
@@ -77,7 +78,13 @@ async def generate_plan(analysis_result: Analysis, backup_database: bool) -> Upg
         PreUpgradeStep(
             description="Verify that all OpenStack applications are in idle state",
             parallel=False,
-            coro=analysis_result.model.wait_for_idle(timeout=11, idle_period=10),
+            coro=analysis_result.model.wait_for_idle(
+                # NOTE (rgildein): We need to DEFAULT_TIMEOUT so it's possible to change if
+                #                  a network is too slow, this could cause an issue.
+                timeout=DEFAULT_TIMEOUT + 1,
```

Comment:
Instead of simply setting it to `DEFAULT_TIMEOUT + 1`, can we do `timeout=max(DEFAULT_TIMEOUT + 1, 11)`? This prevents error in case user sets a DEFAULT_TIMEOUT value smaller than 10

Reply:
No actually I meant `max` because if `DEFAULT_TIMEOUT` is `5`, then we will have `timeout = 5+1, idle_period = 10`, which will always result in failure.